iambic/plugins/v0_1_0/aws/models.py

```diff
@@ -299,6 +301,11 @@ class AWSAccount(ProviderChild, BaseAWSAccountAndOrgModel):
         description="The role arn to assume into when making calls to the account",
         exclude=True,
     )
+    organization_account: bool = Field(
+        False, description="if this is an organization account"
+    )
+    organization: Optional[AWSOrganization] = Field(None)
```

Comment:
These appear in our schema, I think we should provide descriptions to these fields